src/App/Pages/Home.elm

```diff
@@ -15,22 +15,22 @@ import List
 import List.Extra
 
 
-view : Model -> Html Msg
-view model =
+view : Model -> WebData (List Article) -> WebData (List Recipe) -> WebData (List Recipe) -> Html Msg
+view model promotedArticles promotedRecipes recipes =
     div []
-        [ viewPromotedContent model
+        [ viewPromotedContent promotedArticles promotedRecipes
         , viewCurrentMonthIssue model
         , viewCookMenu model
-        , viewRecipes model
+        , viewRecipes recipes
         ]
 
 
-viewPromotedContent : Model -> Html Msg
-viewPromotedContent model =
+viewPromotedContent : WebData (List Article) -> WebData (List Recipe) -> Html Msg
```

Comment:
Here's where an `ItemManager` would have come handy. You are now dealing with `WebData` that you constantly have to unwrap. If you had an `ItemManager` in the middle, it would deal with fetching and with [unwraping](https://github.com/Gizra/drupal-elm-starter/blob/f93745bfcab3e3d28a38ff20838d97482aed1eea/client/src/elm/ItemManager/View.elm#L22-L30) it before handing it down